packages/nimble-components/src/number-field/specs/IxD.md

```diff
@@ -0,0 +1,146 @@
+# Number Field Updates (IxD)
+
+## Overview
+
+### Background
+
+- [Bring number-column formatting options to nimble-number-field](https://github.com/ni/nimble/issues/1855)
+- [Number IxD (Figma)](https://www.figma.com/design/r2yGNQNVFdE7cBO9CyHmQx/Nimble---IxD?node-id=1603-3464)
+
+The `nimble-number-field` leverages
+[fast-foundation](https://github.com/ni/fast/blob/archives/fast-element-1/packages/web-components/fast-foundation/src/number-field/number-field.ts)
+for its behavior, so we've inherited much of the behavior, including max/min
+validation and localization.
+
+For supported languages (English, French, German, Japanese, Chinese), the
+_decimal separator_ does not align with user expectations in German. For
+example, `1000000.01`
+[should](https://randombits.dev/articles/number-localization/locale-list) render
+as `1000000,01` in German but currently renders as `1000000.01` across all
+locales.
+
+Additionally, clients have requested the ability to define max/min values for
+the `nimble-number-field` to support form validation without automatically
+coercing user-entered values to fit within the specified range.
+
+In comparison, the HTML `<input type="number">` natively supports locale-aware
+_decimal separators_ and does not coerce values to the min/max limit.
+
+This IxD proposes updates to the Number Field to natively support localization,
```

Comment:
As we discussed, some of these may not belong in the number field if we're not ready to do the API design to try for maximum consistency with other component APIs and minimum churn as new formatting requirements arise. If we're waiting for the HLD to decide that then we should make this requirement (and maybe other places like the document title) more general, not specify that the API goes in the existing number field.